src/main/java/edu/usf/cutr/gtfsrtvalidator/api/resource/GtfsRtFeed.java

```diff
@@ -428,6 +434,19 @@ public void updateSessionData(
 
         session.saveOrUpdate(sessionModel);
         GTFSDB.commitAndCloseSession(session);
+        session = GTFSDB.initSessionBeginTrans();
+        List<SessionModel> sessionModels = session.createQuery(" FROM SessionModel WHERE sessionEndTime = 0 and rtFeedId = " + sessionModel.getGtfsRtFeedModel().getGtfsRtId()).list();
+        boolean endFeedMonitoring=true;
+        l1:for(SessionModel s : sessionModels){
+            if(s.getSessionEndTime()==0){
+                endFeedMonitoring=false;
+                break l1;
```

Comment:
I don't believe you need the label here to break out of the `FOR` loop - a simple `break;` should work.  I'd prefer that to keep things as simple as possible.